client/src/redux/prop-types.ts

```diff
@@ -24,10 +25,7 @@ export const ChallengeNode = PropTypes.shape({
   challengeType: PropTypes.number,
   dashedName: PropTypes.string,
   description: PropTypes.string,
-  files: PropTypes.shape({
-    indexhtml: FileType,
-    indexjs: FileType
-  }),
+  challengeFiles: PropTypes.array,
```

Comment:
Well, yes. But, I hope to use the already made `ChallengeFile[]` type we have in `prop-types`, when components are migrated. That is, it would not make the migration easier to keep `PropTypes.shape()`, because we already have a type for `challengeFiles` just waiting to be used.